e2e/config.yaml.sample

```diff
@@ -7,6 +7,13 @@ repo:
   url: "https://github.com/RamenDR/ocm-ramen-samples.git"
   branch: main
 
+# Namespace configurations for RamenDR.
```

Comment:
This is not only ramendr, but more about different k8s distros (k8s, openshift). We can add a note that the fault values are good for ramen testing environment.

Keeping this documented for now is good, this will allow supporting any cluster quickly. Once we get this working on openshfit we need to make this configuration hidden and resolve the right name automatically by detecting openshift clusters. We already have code for it.